src/apps/ociobakelut/main.cpp

```diff
@@ -158,58 +174,109 @@ int main (int argc, const char* argv[])
+            config = editableConfig;
         }
-        if(!shaperspace.empty())
+        else if(groupTransform->getTransform(0)->getTransformType() == OCIO::TransformType::TRANSFORM_TYPE_DISPLAY_VIEW)
         {
-            std::cerr << "\nERROR: --shaperspace is not allowed when using --lut\n\n";
-            std::cerr << "See --help for more info." << std::endl;
-            return 1;
-        }
+            if(inputspace.empty())
+            {
+                std::cerr << "\nERROR: --inputspace is required when using --displayview\n\n";
+                std::cerr << "See --help for more info." << std::endl;
+                return 1;
+            }
+            if(!outputspace.empty())
+            {
+                std::cerr << "\nERROR: --outputspace is not allowed when using --displayview\n\n";
+                std::cerr << "See --help for more info." << std::endl;
+                return 1;
+            }
+            if(view.empty())
+            {
+                std::cerr << "\nERROR: Specifying the view is required when using --displayview\n\n";
+                std::cerr << "See --help for more info." << std::endl;
+                return 1;
+            }
+            if(display.empty())
+            {
+                std::cerr << "\nERROR: Specifying the display is required when using --displayview\n\n";
+                std::cerr << "See --help for more info." << std::endl;
+                return 1;
+            }
 
-        OCIO::ConfigRcPtr editableConfig = OCIO::Config::Create();
+            // Creating a new config
+            OCIO::ConfigRcPtr editableConfig = OCIO::Config::Create();
 
-        OCIO::ColorSpaceRcPtr inputColorSpace = OCIO::ColorSpace::Create();
-        inputspace = "RawInput";
-        inputColorSpace->setName(inputspace.c_str());
-        editableConfig->addColorSpace(inputColorSpace);
+            //Setting up input ColorSpace
+            OCIO::ColorSpaceRcPtr inputColorSpace = OCIO::ColorSpace::Create();
+            inputColorSpace->setName(inputspace.c_str());
+            editableConfig->addColorSpace(inputColorSpace);
 
-        OCIO::ColorSpaceRcPtr outputColorSpace = OCIO::ColorSpace::Create();
-        outputspace = "ProcessedOutput";
-        outputColorSpace->setName(outputspace.c_str());
+            //Setting up the output ColorSpace
+            OCIO::ColorSpaceRcPtr outputColorSpace = OCIO::ColorSpace::Create();
+            outputspace = "ProcessedOutput";
+            outputColorSpace->setName(outputspace.c_str());
 
-        outputColorSpace->setTransform(groupTransform,
-            OCIO::COLORSPACE_DIR_FROM_REFERENCE);
+            // Obtaining the transform and applying to outputColorSpace
+            OCIO::DisplayViewTransformRcPtr t = std::dynamic_pointer_cast<OCIO::DisplayViewTransform>(groupTransform->getTransform(0));
+            t->setSrc(inputspace.c_str());
+            outputColorSpace->setTransform(t, OCIO::COLORSPACE_DIR_FROM_REFERENCE);
```

Comment:
Huh?  It doesn't seem like this would work.  The transform is in the user's config.